source/_integrations/keymitt_ble.markdown

```diff
@@ -0,0 +1,62 @@
+---
+title: MicroBot Push
+description: Instructions on how to set up the MicroBot Push.
+ha_category:
+  - Switch
+ha_release: ???
+ha_iot_class: Assumed State
+ha_codeowners:
+  - '@spycle'
+ha_domain: keymitt_ble
+ha_bluetooth: true
+ha_platforms:
+  - switch
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+This integration allows you to locally control a MicroBot Push (previously manufactured by Naran but now under the Keymitt brand).
+
+### Prerequisites
+
+In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant. A Naran/Keymitt hub is not required.
+
+The device will need to be in pairing mode before adding to Home Assistant. To reset the MicroBot Push, turn it off then back on and immediately hold the push button while the LED is red. After approximately 5 seconds the LED will flash rapidly at which point release the button. The LED will blink blue if pairing mode has been successfully activated.
+
+If you have multiple devices, you will need to know the BTLE MAC address of your device to tell them apart. 
+
+Please note, devices cannot remain paired to the MicroBot app for this integration to function. They will be paired to Home Assistant exclusively.
+
+{% include integrations/config_flow.md %}
+
+### Supported Devices
+
+This Integration is for the MicroBot Push only. The Keymitt lock is not supported.
+
+### Service `keymitt_ble.calibrate`
+
+The Calibration service will locally set the MicroBot Push depth, duration, and mode.
+
+Please note: The push arm will extend or retract (dependant on the mode set) after the service call is invoked. The mode and depth will be demomstrated but not the duration. The setting is however stored and can be confirmed by manually operating the device.
+
+| Service Data Attribute | Required | Description                                                                                   |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------- |
+| `entity_id`            | yes      | Entity ID of the MicroBot Push to calibrate.                                                  |
```

Comment:
Isn't this an entity service? (Thus `entity_id` isn't required, one could target areas, devices or everything alternatively).